src/com/projetoFinalModulo1/navalBattle/auxClass/Board.java

```diff
@@ -63,70 +61,14 @@ public void setEmptyPositions(){
         Arrays.fill(position, " ");
     }
 }
+
     private String getLetter(int index){
-        switch (index){
-            case 0:
-                return "A";
-            case 1:
-                return "B";
-            case 2:
-                return "C";
-            case 3:
-                return "D";
-            case 4:
-                return "E";
-            case 5:
-                return "F";
-            case 6:
-                return "G";
-            case 7:
-                return "H";
-            case 8:
-                return "I";
-            case 9:
-                return "J";
-            default:
-                return " ";
-        }
+        return Math.abs(index) < 10 ? this.LETTERS.substring(index, index + 1) : " ";
     }
 
     private int getNumber(String index){
-        switch (index){
-            case "A":
-            case "a":
-                return 0;
-            case "B":
-            case "b":
-                return 1;
-            case "C":
-            case "c":
-                return 2;
-            case "D":
-            case "d":
-                return 3;
-            case "E":
-            case "e":
-                return 4;
-            case "F":
-            case "f":
-                return 5;
-            case "G":
-            case "g":
-                return 6;
-            case "H":
-            case "h":
-                return 7;
-            case "I":
-            case "i":
-                return 8;
-            case "J":
-            case "j":
-                return 9;
-            default:
-                return 99;
-        }
+        return index.contains(index.toUpperCase()) ? this.LETTERS.indexOf(index.toUpperCase()) : 99;
```

Comment:
Ótima simplificação, apenas ficou faltando a retificação do this.LETTERS no lugar do index.